examples/plugins/writing-plugins.rst

```diff
@@ -0,0 +1,131 @@
+       implementation defined in ``PluginFileStore``. The full tracking URI is passed to the ``PluginFileStore`` constructor.
+     - `FileStore <https://github.com/mlflow/mlflow/blob/master/mlflow/store/tracking/file_store.py>`_
+
+   * - Plugins for defining artifact read/write APIs like ``mlflow.log_artifact``, ``MlflowClient.download_artifacts`` for a specified
+       artifact URI scheme (e.g. the scheme used by your in-house blob storage system).
+     - mlflow.artifact_repository
+     - The entry point value (e.g. ``mlflow_test_plugin:PluginLocalArtifactRepository``) specifies a custom subclass of
+       `mlflow.store.artifact.artifact_repo.ArtifactRepository <https://github.com/mlflow/mlflow/blob/master/mlflow/store/artifact/artifact_repo.py#L12>`_
+       (e.g., the `PluginLocalArtifactRepository class <https://github.com/mlflow/mlflow/blob/master/tests/resources/mlflow-test-plugin/mlflow_test_plugin/__init__.py#L18>`_
+       within the ``mlflow_test_plugin`` module).
+
+       The entry point name (e.g. ``file-plugin``) is the artifact URI scheme with which to associate the custom ArtifactRepository implementation.
+       In the example above, who install the plugin & log to a run whose artifact URI is of the form "file-plugin://<path>" will use the
+       custom ArtifactRepository implementation defined in ``PluginLocalArtifactRepository``.
+       The full artifact URI is passed to the ``PluginLocalArtifactRepository`` constructor.
+     - `LocalArtifactRepository <https://github.com/mlflow/mlflow/blob/master/mlflow/store/artifact/local_artifact_repo.py>`_
+
+
+   * - Plugins for specifying custom context tags at run creation time, e.g. tags indicating the git repo
+       the run is associated with.
+     - mlflow.run_context_provider
+     - The entry point name is unused. The entry point value (e.g. ``mlflow_test_plugin:PluginRunContextProvider``) specifies a custom subclass of
+       `mlflow.tracking.context.abstract_context.RunContextProvider <https://github.com/mlflow/mlflow/blob/master/mlflow/tracking/context/abstract_context.py#L4>`_
+       (e.g., the `PluginRunContextProvider class <https://github.com/mlflow/mlflow/blob/master/tests/resources/mlflow-test-plugin/mlflow_test_plugin/__init__.py#L23>`_
+       within the ``mlflow_test_plugin`` module) to register.
+     - `GitRunContext <https://github.com/mlflow/mlflow/blob/master/mlflow/tracking/context/git_context.py#L36>`_,
+       `DefaultRunContext <https://github.com/mlflow/mlflow/blob/master/mlflow/tracking/context/default_context.py#L41>`_
+
+   * - Plugins for overriding definitions of model registry APIs like mlflow.register_model.
+     - mlflow.model_registry_store
+     - **Note**: The model registry is in beta (as of MLflow 1.5), so APIs are not guaranteed to be stable & model-registry plugins may break in the
+       future.
```

Comment:
**Note**: The model registry is in beta (as of MLflow 1.5), so APIs are not guaranteed to be stable & model-registry plugins may break in the future.

->

.. note: The model registry is in Beta (as of MLflow 1.5), so APIs are not guaranteed to be stable and model-registry plugins may not be supported in future releases.